xml/System.Net.Http/HttpClient.xml

```diff
@@ -125,6 +109,74 @@ If this change is undesirable, on Windows you can still use <xref:System.Net.Htt
+ 
+If an app using <xref:System.Net.Http.HttpClient> and related classes in the <xref:System.Net.Http> namespace intends to download large amounts of data (50 megabytes or more), then the app should stream those downloads and not use the default buffering. If the default buffering is used the client memory usage will get very large, potentially resulting in substantially reduced performance.  
+  
+### Thread Safety
+
+ The following methods are thread safe:  
+  
+1.  <xref:System.Net.Http.HttpClient.CancelPendingRequests%2A>  
+  
+2.  <xref:System.Net.Http.HttpClient.DeleteAsync%2A>  
+  
+3.  <xref:System.Net.Http.HttpClient.GetAsync%2A>  
+  
+4.  <xref:System.Net.Http.HttpClient.GetByteArrayAsync%2A>  
+  
+5.  <xref:System.Net.Http.HttpClient.GetStreamAsync%2A>  
+  
+6.  <xref:System.Net.Http.HttpClient.GetStringAsync%2A>  
+  
+7.  <xref:System.Net.Http.HttpClient.PostAsync%2A>  
+  
+8.  <xref:System.Net.Http.HttpClient.PutAsync%2A>  
+  
+9.  <xref:System.Net.Http.HttpClient.SendAsync%2A>  
+  
+### Proxies
+
+By deafult HttpClient will read proxy configuration from environment variables or user / system settings depending on the platform being used. This can be changed by passing a <xref:System.Net.WebProxy> or <xref:System.Net.IWebProxy> to, in order of precedence:
+
+* The <xref:System.Net.Http.HttpClientHandler.Proxy> property on a HttpClientHandler passed in during HttpClient construction
+* The <xref:System.Net.http.HttpClient.DefaultProxy%2A> static property (affects all instanes)
+
+The proxy can be disabled using <xref:System.Net.Http.HttpClientHandler.UseProxy>. The default configuration for Windows users is to try and detect a proxy using network discovery which can be slow. For high througput applications where its known that a proxy is not required, the proxy should be disabled.
+
+### Timeouts
+
+HttpClient.Timeout can be used to set a default timeout for all Http requests from the HttpClient instance. The timeout only applies to the xxxAsync methods that cause a request/response to be initiated. If the timeout is reached, the <xref:System.Threading.Tasks.Task%601> for that request will be cancelled.
+
+Some Additional timeouts can be set if you construct the HttpClient passing in a <xref:System.Net.Http.SocketsHttpHandler> instance:
+
+| **Property** | **Description**|
+| ------------ | -------------- |
+| <xref:System.Net.Http.SocketsHttpHandler.ConnectTimeout> | Specifies a timeout that is used when a request requires a new TCP connection to be created. If the timout occurs the request <xref:System.Threading.Tasks.Task%601> is cancelled. |
+| <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionLifetime> | Specifies a timeout to be used for each connection in the connection pool. If the connection is idle, it is immediately closed, otherwise at the end of the current request. |
```

Comment:
```suggestion
| <xref:System.Net.Http.SocketsHttpHandler.PooledConnectionLifetime> | Specifies a timeout to be used for each connection in the connection pool. If the connection is idle, the connection is immediately closed; otherwise, the connection is closed at the end of the current request. |
```